LibraBFT/ImplShared/Consensus/Types/EpochIndep.agda

```diff
@@ -560,19 +591,6 @@ module LibraBFT.ImplShared.Consensus.Types.EpochIndep where
     UnexpectedRound       : Round → Round →      VoteReceptionResult
     VRR_TODO              :                      VoteReceptionResult
 
-  data Output : Set where
```

Comment:
I guess you intended to move this somewhere else, which makes sense, but it doesn't seem to be anywhere now.

Reply:
It has been added to `LibraBFT.ImplShared/Consensus/Types/MetaEpochIndep.agda`